src/types.ts

```diff
@@ -1,4 +1,19 @@
+/*
+  Tickets usually contain the following keys, but depending on the template may contain any key:
+  * id?: number | string,
+  * title?: string,
+  * description?: string,
+  * type?: string,
+  * url?: string,
+  * branch?: string,
+  * commit?: string,
```

Comment:
What's the reason for making this a comment and not a proper type specification? 🤔

Reply:
it was a type spec first, but then I realized that you can add *any* value to a ticket and, depending on your template, that is totally fine. Didn't know how to say

> these are good values, but totally optional and only in the defaults, really from a type-validation-perspective you can put anything in there.

Reply:
@pmeinhardt do you have a better idea on how to solve this? or is this good enough?

Reply:
Ah, right. Because this typespec would pin it to what we have in Tickety-Tick and this is a generic package now. Makes sense. I'm wondering though (I haven't written much TypeScript so far) whether `object` would be a slightly better type than `any` for `Ticket`?

Reply:
Not sure. I'd be fine with keeping it as it is. 🙂

Reply:
`object` is a good idea, will try that and (if it works) merge this PR

Reply:
oh, I think i found a good solution (declaring an object having string-keys only and number or string values). This is way more specific than `any`. thanks paul for holding my hand on this journey :)